roles/system/tasks/main.yml

```diff
@@ -17,14 +17,14 @@
   when: ansible_lsb.id == "Debian"
   tags: ['master','update']
 
-- name: Run apt-get update  
-  shell: apt-get update ; echo
-  register: result
+- name: Update APT cache, not more than once every 8 hours
+  apt: 
+    update_cache: yes 
+    cache_valid_time: 28800
   tags: ['master','custom','update']
 
 - name: Fix broken package
   shell: dpkg --configure -a
-  when: "'dpkg --configure -a' in result.stderr"
```

Comment:
Is there a reason to play this task at every run ? 
`when: "'dpkg --configure -a' in result.stderr"` was there to play this task only when `shell: apt-get update ; echo` crashed